### PR TITLE
snapcraft.yaml: Adds minio to LXD package

### PR DESCRIPTION
**Pull request by tomponline**
Signed-off-by: Thomas Parrott <thomas.parrott@canonical.com>

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -508,6 +508,32 @@ parts:
...
+      - go
+    override-pull: |
+      snapcraftctl pull
+    override-build: |
+      set -ex
+
+      # Setup build environment
+      export GOPATH=$(realpath ./.go)
+      export CGO_CFLAGS="-I${SNAPCRAFT_STAGE}/include/ -I${SNAPCRAFT_STAGE}/usr/local/include/"
+      export CGO_LDFLAGS="-L${SNAPCRAFT_STAGE}/lib/ -L${SNAPCRAFT_STAGE}/usr/local/lib/"
```
Comment (stgraber) [inline snapcraft.yaml]: Those shouldn't be needed as minio doesn't need linking against dqlite, liblxc or similar.

Comment (tomponline) [reply snapcraft.yaml]: Ok thanks, im going to pin the version tag too.

Comment (tomponline) [reply snapcraft.yaml]: Fixed.

**Comment by tomponline** [comment]
Oh just need to fix the commit sign off as new repo.

**Pull request merged**